spec/features/likes_feature_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'spec_helper'
+
+feature 'Likes' do
+
+  before do
+    user = create :user
+    log_in(user)
+    click_link 'New Post'
+    attach_file('picture[image]', File.join(Rails.root,'spec',"files", 'images', 'duck.jpg'))
```

Comment:
Metrics/LineLength: Line is too long. [93/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)